url-plugins/docs/UrlFetch-realtimesource.md

```diff
@@ -0,0 +1,15 @@
+# URL Fetch Real-time Source
+
+This is a real-time source that will fetch data from a specified URL at a given interval and pass the results to the
```

Comment:
please follow the format used in core-plugins, with description, use case, properties, and example sections